docs/manual/installation/system-requirements.de.md

```diff
@@ -18,6 +18,9 @@ empfohlen, diese immer zu verwenden.
 
 - **PHP:** Version 7.3+ (neueste Patch-Version)
 - **MySQL:** Version 5.7+ oder gleichwertiger **MariaDB** Server
+{{% notice warning %}}
+Derzeit besteht eine Inkompatibilität mit MySQL Version **8.0.14+**.
```

Comment:
`MEMBER` was added in `8.0.17`, not `8.0.14`. See https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Reply:
There are 3 keywords in question, see https://github.com/doctrine/dbal/pull/3674#issue-315887966

Reply:
Only the `MEMBER` keywords is currently relevant within Contao - unless an extension uses `LATERAL` somewhere without quotes as well.